editor/src/clj/editor/gui.clj

```diff
@@ -2943,16 +2943,19 @@
       :nodes node-descs)))
 
 (defn- build-pb [resource dep-resources user-data]
-  (let [def (:def user-data)
-        pb  (:pb user-data)
-        pb  (if (:transform-fn def) ((:transform-fn def) pb) pb)
-        pb  (reduce (fn [pb [label resource]]
-                      (if (vector? label)
-                        (assoc-in pb label resource)
-                        (assoc pb label resource)))
-                    pb (map (fn [[label res]]
-                              [label (resource/resource->proj-path (get dep-resources res))])
-                            (:dep-resources user-data)))]
+  (let [pb (transduce
+             (map (fn [[label build-resource]]
+                    {:pre [(or (nil? build-resource) (workspace/build-resource? build-resource))]}
+                    (let [fused-build-resource (get dep-resources build-resource)
+                          fused-build-resource-path (resource/resource->proj-path fused-build-resource)]
+                      (pair label fused-build-resource-path))))
+             (completing
+               (fn [pb [label fused-build-resource-path]]
+                     (if (vector? label)
+                       (assoc-in pb label fused-build-resource-path)
+                       (assoc pb label fused-build-resource-path))))
+             (:pb user-data)
+             (:dep-resources user-data))]
```

Comment:
I did the work to prepare Gui resources to `:allow-unloaded-use`, but I've disabled it for now since saving will strip out all imported `:template-node-child` `NodeDescs` from referencing scenes.